am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -165,7 +169,13 @@ public void revokeResourceAccess(@NotNull @Valid ExplicitAccessMetadata accessMe
         + "and permissions {{result.access.permissions}}")
     public FilteredResourceEnvelope filterResource(@NotBlank String userId,
                                                    @NotEmpty Set<@NotBlank String> userRoles,
-                                                   @NotNull @Valid Resource resource) {
+                                                   @NotNull @Valid Resource resource,
```

Comment:
@Auditlog needed may be like this "security classifications '{{result.securityClassifications}}"